.ci/rocky_db_seed.json

```diff
@@ -20,9 +20,7 @@
     "pk": 1,
     "fields": {
       "name": "test-organization",
-      "code": "_dev",
-      "signal_username": null,
-      "signal_group_id": null
+      "code": "_dev"
```

Comment:
Do we still need this _dev in here? I thinik the onboarding flow now creates the first organization.